File: Jan_Basic_Programming/Assignments/C/32.c
#include <stdio.h>
int main()
{
    char str[50];
    int i = 0, words = 1;

    printf("Enter the Sentence : ");
    scanf(" %[^\n]", str);

    while (str[i] != '\0')
    {
        if ((int)str[i] == 32 && (int)str[i + 1] != 32)
        {
            words++;
        }
        i++;
    }

    printf("Words in Sentence : %d", words);
    return 0;
}

File: Mar_PHP/Assignments/BasicPrograms/32.php
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Word Count in a Sentence</title>
</head>
<body>
    <h2>Word Count in a Sentence</h2>
    <form method="post">
        Enter a sentence: <input type="text" name="sentence" required>
        <input type="submit" name="submit" value="Count Words">
    </form>

    <?php
    if (isset($_POST['submit'])) {
        $sentence = $_POST['sentence'];
        $word_count = str_word_count($sentence);

        echo "<p>Number of words in the sentence: $word_count</p>";
    }
    ?>
</body>
</html>
